src/utils/parse-arguments-into-options.ts

```diff
@@ -39,6 +39,9 @@ export async function parseArgumentsIntoOptions(rawArgs: Args): Promise<RawOptio
 
       "--dev": Boolean,
 
+      "--solidity-framework": solidityFrameworkHandler,
+      "-f": "--solidity-framework",
```

Comment:
Should we use `-sf` ? 😅 

-f seems like selecting foundry directly

Reply:
`-sf` doesn't work. `arg` lib understands it as `-s -f`. As I remember I wanted to use `-sf` too :) but then changed it

Reply:
[df1255b](https://github.com/scaffold-eth/create-eth/pull/58/commits/df1255b5c6295217c590d5b98cdd9340db08683c) makes sense, thanks! Changed